scripts/data/format_args.py

```diff
@@ -110,17 +110,34 @@ def format_item(item):
     return format_item(args)
 
 
-def format_args():
+def format_args(input_file, execute_script):
     """Reads arguments from JSON file and prints formatted result.
     Expects a single CLI argument containing file path.
     Output is compatible with the Scarb runner arguments format.
     """
-    if len(sys.argv) != 2:
-        raise TypeError("Expected single argument")
-    args = json.loads(Path(sys.argv[1]).read_text())
+    args = json.loads(Path(input_file).read_text())
     res = flatten_tuples(serialize(args))
-    print([res])
+    print([res, 1 if execute_script else 0])
 
 
 if __name__ == "__main__":
```

Comment:
I'd really prefer to keep this script simple in>format>out, are there any reasons the execute flag is better than a feature?

Reply:
Flag is more explicit. I feel features are reserved for low level stuff like a choice of sha256 implementation.

Reply:
And features are not well supported in a multi package project:
![image](https://github.com/user-attachments/assets/bfcf81d1-9d12-4c0d-a14f-f4edaf82156a)